MAINTAINERS.md

```diff
@@ -0,0 +1,16 @@
+# Maintainers
+
+This document lists the current maintainers of Project Copacetic. Maintainers guide project direction, review contributions, and handle releases.
+
+## Active Maintainers
+
+| Name             | GitHub Handle       | Affiliation |
+|------------------|---------------------|-------------|
+| Sertac Ozercan   | [@sozercan](https://github.com/sozercan) | Microsoft | 
+| Ashna Mehrotra     | [@ashnamehrotra](https://github.com/ashnamehrotra) | Microsoft | 
+| Leo Wang     | [@leodewang](https://github.com/leodewang) | Microsoft | 
+| Robbie Cronin     | [@robert-cronin](https://github.com/robert-cronin) | Microsoft | 
+| Joshua Duffney     | [@duffney](https://github.com/duffney) | Microsoft | 
+
```

Comment:
New maintainers should be voted in by existing maintainers as outlined in a governance doc, which is missing. e.g. https://github.com/open-policy-agent/opa/blob/main/GOVERNANCE.md#changes-in-maintainership

 New maintainers need to show data for contributions to meet criteria of becoming a maintainer, which should be documented in the contributor ladder doc (also need to add) e.g. https://github.com/getporter/porter/blob/main/CONTRIBUTION_LADDER.md

Reply:
Yep, I'll be taking a look at the governance doc, and Josh for the contributor ladder doc

Reply:
I'll have a PR ready for the ladder soon. :)